the_game.pl

```diff
@@ -72,20 +79,29 @@
 	).
     
 display_help :-
-	write("Type a command followed by a period to perform an action."), nl,
-	write("help - show this instruction"), nl,
-	write("quit - quit the game"), nl,
-	write("show - describe the current location"), nl,
-	write("north - go north"), nl,
-	write("east - go east"), nl,
-	write("south - go south"), nl,
-	write("west - go west"), nl,
-	write("take - take an item in the current location"), nl,
-	write("talk - talk to a person in the current location"), nl,
-	write("stats - to show character statistics"), nl,
+	println("Type a command followed by a period to perform an action."),
+	println("help - show this instruction"),
+	println("quit - quit the game"),
+	println("show - describe the current location"),
+	println("north - go north"),
+	println("east - go east"),
+	println("south - go south"),
+	println("west - go west"),
+	println("take - take an item in the current location"),
+	println("talk - talk to a person in the current location"),
+	println("stats - to show character statistics"),
+	println("char - to show hero characteristics"),
+	println("achiev - to show list of collected achivements"),
+	println("quests - shows the list of quests"),
+	println("look - to check where we can move"),
 	try((
 		has(hero, location_map),
-		write("map - show the map of the area"), nl
+		println("map - show the map of the area")
+	)),
+	try((
```

Comment:
It would be nice to decide if we want to list here only the currently available actions or all actions at all. "Open" seems quite a general action while the exception with the map was in fact a seed of the convention to type the name of an item the hero has in his inventory to use it.